cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/file/FileSetDataset.java

```diff
@@ -208,8 +207,9 @@ public String getInputFormatClassName() {
 
   @Override
   public Map<String, String> getInputFormatConfiguration(Iterable<? extends Location> inputLocs) {
-    ImmutableMap.Builder<String, String> config = ImmutableMap.builder();
```

Comment:
the reason for this change is that the runtime arguments may override properties, that is, they will contain the same key that was already added from the properties. With ImmutableMap.Builder, that would fail, because it throw exceptions on repeated keys. We can just use a HashMap here, because the result does not need to be immutable.